kroxylicious-runtime/src/test/java/io/kroxylicious/proxy/config/NamedRangeTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.config;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+class NamedRangeTest {
+
+    @Test
+    void contains() {
+        assertThat(new NamedRange("myrange", 1, 3).values()).containsExactlyInAnyOrder(1, 2, 3);
+    }
+
+    @Test
+    void singleton() {
+        assertThat(new NamedRange("myrange", 1, 1).values()).containsExactlyInAnyOrder(1);
+    }
+
+    @Test
+    void endBeforeStartIllegal() {
+        assertThatThrownBy(() -> new NamedRange("myrange", 1, 0))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("end of range: 0 is before start of range: 1");
+    }
+
+    @Test
+    void toStringComposition() {
+        assertThat(new NamedRange("foo", 1, 2))
+                .hasToString("NamedRange{name='foo', start=1, end=2}");
+    }
```

Comment:
Is this really worth testing?

Reply:
Agreed.  The previous incarnation used range notation.. [1,3) so it was kind of justified.